tests/functions/test_origin_response.py

```diff
@@ -60,7 +60,7 @@ def test_origin_response_valid_headers(original_uri, want_digest):
             }
         ]
 
-    response = OriginResponse(conf_file=CONF_PATH).handler(event, context=None)
+    response = OriginResponse(conf_file=TEST_CONF).handler(event, context=None)
```

Comment:
The CONF_PATH => TEST_CONF change isn't necessary but doesn't hurt anything either.